### ISSUE TITLE
You can drag the "yellow" sphere outside of the "Mono Panner" and it will be "lost"

### ISSUE THREAD

**Issue by hems** [EnvelopSound/EnvelopForLive#95]
If you click and drag the yellow circle to outside of the max4live device boundaries it will be "left outside" of the canvas, like this:

![image](https://user-images.githubusercontent.com/27327/208211173-aaab0144-6576-4404-9e6c-d9350b978d61.png)

i wonder if it makes sense to let the user drag out the boundaries?

**Comment by mcslee** [comment]
Yeah this should be restricted. I will see if this can be contained by the underlying `ambimonitor~` device. FYI in the meantime adjusting any of the Azim/Elev/Radius knobs will bring the yellow dot back.
